narratives/ncov_sit-rep_ja_2020-03-20.md

```diff
@@ -255,10 +255,10 @@ COVID-19 が突然爆発したように見えるかもしれません。
 <br>
 - 多くの地域で地域伝搬の証拠が見られます。これを超えて、過去の伝搬により、今後数週間で明らかになる多くの地域流行がすでに進行中であることが予想されます。  
 <br>
-- 社会的距離をとおして地域の発生を制御することは:  
-  - #FlattenTheCurve （流行曲線の抑制）を使用して、医療システムに対する感染者の集中を防ぎます。  
+- 社会的距離をとおして地域の感染流行を制御することは:  
+  - #FlattenTheCurve （流行曲線の抑制して）、医療システムに対する感染者の集中を防ぎます。  

```

Comment:
for the following line, I prefer the previous one. Except this point, others are pretty fine and actually adding fluency.

> - #FlattenTheCurve （流行曲線の抑制）を使用して

Reply:
rollbacked. Thanks for rapid reviewing!